<launch>
    
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" > 
    	<arg name="port" value="9090"/>
  	</include>
  	<include file="$(find clm_ros_wrapper)/launch/clm_ros_wrapper.launch" />
    
    <!-- Global variables here -->
    <param name="/sar/global/_p_sar_core_dir" type="str" value="$(find sar_core)" />
    <param name="/sar/global/_child_name" type="str" value="Eliza" />
    <param name="/sar/global/_guardian_name" type="str" value="Paul" />

  	<node pkg="sar_core" type="user_tracking_fusion.py" name="user_tracking_fusion" output="screen">
    		<param name="flag_fusion_mode" type="str" value="left" />
  	</node>
  	<node pkg="sar_core" type="user_monitor.py" name="user_monitor" output="screen">
  	    <param name="test_flag_face_following" type="bool" value="false" />
  	    <param name="test_flag_ja_seeking" type="bool" value="false" />
  	</node>

    <node pkg="sar_core" type="vja_monitor.py" name="vja_monitor" output="screen" />
    <node pkg="sar_robot_translation" type="robot_translation_node.py" name="robot_translation_node" output="screen" />
</launch>
